docs/source/examples.rst

```diff
@@ -6,6 +6,19 @@ Usage Examples
 .. toctree::
    :maxdepth: 1
 
+.. _example_dependencies:
+Example Dependencies
+------------------------
+
+Some examples may use additional dependecies. These dependencies are listed in `examples_requirements.txt <https://github.com/lanl/scico/blob/main/examples/examples_requirements.txt>`_.
```

Comment:
The "may" is redundant given the presence of the "some" - I suggest removing it. Also, there's a typo in "dependecies", and perhaps change "... dependecies. These are ..." to "... additional dependencies, which are ...".

Reply:
Fixed it.